packages/connect/src/relay/index.ts

```diff
@@ -232,20 +278,17 @@ class Relay {
       handShakeResult.counterparty,
       conn.connection.remotePeer,
       handShakeResult.stream
-    ) as any
+    )
 
-    let upgraded: Connection
     try {
-      upgraded = (await this.libp2p.upgrader.upgradeInbound(newConn)) as any
+      await this.libp2p.upgrader.upgradeInbound(newConn as any)
```

Comment:
automatically emit an internal event in libp2p, so no need to return the connection object